### DISCUSSION TITLE
How can I set up Jiten on a device with low RAM (less than 1GB)?

### DISCUSSION THREAD

**Discussion by maxigaz** [obfusk/jiten#157] [Q&A] [answered]
I have a Moto G (osprey) with 1GB of RAM running postmarketOS. When I run `jiten setup` for the first time and `writing sentences` reaches 100%, RAM usage starts to increase until the device becomes completely unresponsive (and needs to be hard reset).

I have already installed and set up Jiten on a different phone with 3GB of RAM in Android and Termux (and also on my Linux x86_64 PC with 16GB of RAM), so one possible workaround I have in mind is to copy the compiled data files from there to the Moto G. However, I'm not sure where they are located and whether they are compatible across these two systems.

Any suggestions are welcome.

**Comment by obfusk** [comment, marked as answer]
`jiten setup --download` should work if you have internet :)

```
$ jiten setup --help
Usage: jiten setup [OPTIONS]

  Build (or download) sqlite databases.

Options:
  --download  Always download DBs, never build them.
  --help      Show this message and exit.
```

**Reply by maxigaz** [reply]
Thank you for the great answer!

**Reply by obfusk** [reply]
You should also be able to copy the 4 `.sqlite3` files in the `jiten/res` directory from another install.
